test/pageobjects/Confirm.js

```diff
@@ -80,6 +82,10 @@ class Confirm extends BasePage {
     this.uploadedVideo.isDisplayed()
     this.driver.executeScript("arguments[0].play();", this.uploadedVideo)
   }
+
+  async waitForConfirmBtnToBeLocated() {
+    this.driver.wait(until.elementLocated(this.confirmBtnSelector))
```

Comment:
minor, you could also make this into a more concise function that gets reused.
```
async waitForElement(elementCss) {
   this.driver.wait(until.elementLocated(element))
}
```

Reply:
Yes, I was about to suggest the same. It will avoid repetition